component/prometheus/operator/podmonitors/config_gen/config_gen.go

```diff
@@ -0,0 +1,164 @@
+package config_gen
+
+// SEE https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/prometheus/promcfg.go
```

Comment:
Do we want to pin this comment to the specific SHA we copied it from so it's easier to tell if we get out of sync?

Reply:
Ditto in the other file.